### DISCUSSION TITLE
Name this plugin

### DISCUSSION THREAD

**Discussion by creecros** [creecros/Mailmagik#14] [Polls]
Let's do a poll to name it.

**Comment by aljawaid** [comment]
TaskEmailer or EmailConverter

**Reply by creecros** [reply]
Im routing for "KbImapIsFunPlugin" !!!

**Reply by alfredbuehler** [reply]
I'd omit the 'E', we know that it's 'electronic'. ;)  TaskMailer is my thing!

**Reply by creecros** [reply]
I'm kinda diggin EmailMagik, come on, let's make EmailMagik together.

**Reply by creecros** [reply]
Actually MailMagik has a better ring
Goes great with MetaMagik

**Comment by creecros** [comment]
So I must suck, but everytime i tried changing the name, it stops working. I tried three times, and it made no sense. Like I'm clueless as to why I failed.

**Reply by alfredbuehler** [reply]
But where exactly did it fail?

**Reply by aljawaid** [reply]
I created a new branch, let me try... what name did you finalise on?

**Reply by creecros** [reply]
it would fail during scanning mails. 

I was gonna do MailMagik, but I'm fine with whatever.

**Reply by creecros** [reply]
it was as if it no longer wanted to connect and screen would go white after about 2 secs of scanning.

**Reply by aljawaid** [reply]
try this: cfc2a1a5fc7cb691a6128d50cbefcb552ad208a5
but rename the folder too

@creecros let me know if it works, then I will do a PR into the default branch

**Reply by creecros** [reply]
awesome I'll check it out. give me an hour or so, just getting up

**Reply by aljawaid** [reply]
> awesome I'll check it out. give me an hour or so, just getting up
I didnt change any variables or config settings... I think thats where it can get confusing and I need 2 coffees to sift through all that hahaha

**Reply by creecros** [reply]
ya, kept suspecting I was putting in the wrong server info....oh wait...I just had a brain fart.....now that I think about it.

**Reply by creecros** [reply]
ya, that makes sense now....I'm an idiot. I only put in the username and password....the server info placeholders....well, let's just say I never filled that info in.

**Reply by creecros** [reply]
which brings me to...should probably put a error message in for that case scenario...and maybe even a connection failure

**Reply by aljawaid** [reply]
> which brings me to...should probably put a error message in for that case scenario...and maybe even a connection failure

Yes definitely. End user experience is important in these kind of situations.

**Reply by creecros** [reply]
ya, turns out everything was fine, i just pushed the name change i did.

**Reply by aljawaid** [reply]
MailMagik or Mailmagik.... your readme doesnt match what you have done...

- Mailmagik is not really making sense for grammar or naming

**Reply by creecros** [reply]
lol Magik is improper grammer anyway. if thats the logic, it would be Mail Magic. its all good.
